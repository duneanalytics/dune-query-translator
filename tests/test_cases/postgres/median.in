select tx_index, percentile_cont(0.5) within group (order by tx_index) from ethereum.traces where block_number < 100000 group by 1