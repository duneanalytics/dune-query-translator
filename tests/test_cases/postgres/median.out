SELECT
  tx_index,
  APPROX_PERCENTILE(tx_index, 0.5)
FROM ethereum.traces
WHERE
  block_number < 100000
GROUP BY
  1